### DISCUSSION TITLE
How to size properly an image which will appear in the carousel ?

### DISCUSSION THREAD

**Discussion by MozeBaltyk** [razonyang/hugo-theme-bootstrap#973] [Q&A] [answered]
Inside the index.md  of my article :
```md
---
# type: docs 
title: How to Create this Blog
images: [./how-to-create-this-blog/carousel.webp]
---
```

Then I push my image in ./static/how-to-create-this-blog/carousel3.webp

But it appear a little bit squash. 

There is any pixels height that I should respect ?

**Comment by razonyang** [comment, marked as answer]
Could you please provide the image and screenshot to see how it's displaying? I remember it's designed to respect the image ratio.

**Reply by MozeBaltyk** [reply]
![wallpaper-25617401](https://github.com/razonyang/hugo-theme-bootstrap/assets/35733045/7c3bfb5a-be8e-481a-900c-4ccf9fb05f95)

**Reply by MozeBaltyk** [reply]
Of course, if I crop it, it display nice... but I just wanted to know if there is a trick or command to size it properly ? 

Anyway, a big thanks for your theme, which bring all what I was looking for. In a near future, I would like to publish as github page, articles and documentation about sysadmin.  ^^

**Reply by razonyang** [reply]
Glad you like it. Seems the rendered ratio isn't same as original, will check.

![image](https://github.com/razonyang/hugo-theme-bootstrap/assets/17720932/c8f1e153-5957-49f0-b99d-b552967e6b18)

**Reply by razonyang** [reply]
Created an issue #974 for tracking this, I don't have a solution for now.

**Reply by MozeBaltyk** [reply]
I am using git submodule to include in my project : 
[hugo-theme-bootstrap @ 38321ed](https://github.com/razonyang/hugo-theme-bootstrap/tree/38321ed9bb3e68bd6ba47a701a8febb6293ff5b6)

**Reply by razonyang** [reply]
Thanks for the details, confirmed, but have no solution for now 😞.

**Reply by MozeBaltyk** [reply]
No problem, it's alright!
